### PR TITLE
Spring Boot testing HW

### PR DESCRIPTION
**Pull request by aleks-kash**
Create an integration test for at least one endpoint in TodoController
Use Testcontainers
Feel free to use @Sql if required

Advanced:
Setup security in your app
Allow authentication and authorization with JWT
Replace hardcoded userId and changedBy in the previous HW with valid user data received after successful authentication

### CODE REVIEW

**Comment by aleks-kash** [comment]
author: Oleksii Kashtanov

File: src/it/java/org/example/springbootapp/controller/TodoControllerIT.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andReturn();
+
+        // Then
+        TodoResponseDto[] actual = objectMapper.readValue(result.getResponse().getContentAsByteArray(), TodoResponseDto[].class);
+        Assertions.assertEquals(3, actual.length);
+        Assertions.assertEquals(expected, Arrays.stream(actual).toList());
+    }
+}
```
Comment (b-chupika) [inline src/it/java/org/example/springbootapp/controller/TodoControllerIT.java:106]: new line at the EOF is expected. The same for other files

File: src/main/java/org/example/springbootapp/config/SecurityConfig.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    }
+
+    @Bean
+    public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
+        return http
+                .cors(AbstractHttpConfigurer::disable)
+                .csrf(AbstractHttpConfigurer::disable)
+                .authorizeHttpRequests(
+                        auth -> auth
+//                                .requestMatchers("/auth/**")
```
Comment (b-chupika) [inline src/main/java/org/example/springbootapp/config/SecurityConfig.java:41]: no need to push commented code

**Review by b-chupika — APPROVED** [review]
Great job!

**Pull request merged**